README.md

```diff
@@ -70,17 +73,19 @@ This has two benefits:
 > It is best practice for tags for repositories which will be used in GitHub Actions workflows to be prefixed with `v` before the version number, so tags in this repository should start with a `v` prefix too.
 
 
-[community health files]: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file
+[community health files]:  https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file
 
-[reusable-markdownlint]:  https://github.com/PHPCSStandards/.github/blob/main/.github/workflows/reusable-markdownlint.yml
-[markdownlint-cli2]:      https://github.com/DavidAnson/markdownlint-cli2
+[reusable-markdownlint]:   https://github.com/PHPCSStandards/.github/blob/main/.github/workflows/reusable-markdownlint.yml
```

Comment:
I would prefer to remove the extra spaces after the colon rather than adding more. When looking at `git blame` these lines show up unnecessarily.

Reply:
I agree on the negative effect on `git blame`, but this does greatly improve readability of the markdown source and if we're talking consistency, aligned link lists are also used in most other repos in this organisation (including the PHPCS changelogs and such).